generators/app/templates/src/StarterKit/_config/app.json

```diff
@@ -5,6 +5,10 @@
     "DataDirectory": "/data/",
     "LogExceptions": "true",
     "TempDirectory": "/temp/",
-    "DisableGlobalErrorHandling": "false"
+    "DisableGlobalErrorHandling": "false",
+    "RequestLogging": {
+      "IncomingEnabled": true,
```

Comment:
je hebt settings om inkomende en uitgaande logs te enablen, maar in code zie ik regelmatig if(Log.IsEnabled(LogEventLevel.Debug) staan => wil dit dan zeggen dat deze settings enkel iets doen als het log lebvel op debug staat? Ik denk dat hier een keuze gemaakt moet worden: of settings gebruiken, of loggen afhankelijk van andere condities (minimum log level, http codes, ...)

Reply:
@RobLiekens De debug check is voornamelijk om te kijken of er een payload gelogd moet worden, zo stond dat in de logging requirements.
@PeterBrion Gaf later als feedback om misschien ook een setting toe te voegen om de incoming & outgoing logs in z'n geheel te disablen. Vandaar die setting.
We kunnen 1 van de 2 gebruiken uiteraard maar dan is wel de vraag wat daarbij de verwachte functionaliteit is.

Reply:
@PeterBrion Is het misschien niet gebruiksvriendelijker om de payload sowieso mee te loggen als er een fout optreedt? Lijkt me nogal omslachtig om het log level op debug te moeten zetten, te redeployen en dan te wachten tot de fout opnieuw voorvalt (als dat al gebeurt). Bovendien creëert dit een hele hoop nutteloze log content in systemen met hoge load wanneer er getracht wordt om een specifieke fout op te sporen.

Reply:
Lijkt mij een prima voorstel. Nog wel evne ter bevestiging dan, de LogPayloadOnError staat los van de LogPayload? Dus als LogPayloadOnError op true staat logt die altijd bij een foutmelding ongeacht de LogPayload setting?

Reply:
Ja, dat zou ik zo doen.

Als een van beide LogPayload* waardes true is, zou ik die voorrang geven over de andere waarde. Dus bv. bij LogPayload = true en LogPayloadOnError = false worden error payloads toch gelogd omdat LogPayload = true.